lib/vsc/mympirun/mpi/openmpi.py

```
@@ -66,7 +67,20 @@ def pinning_override(self):
         self.log.debug("pinning_override: type %s ", override_type)
 
         ranktxt = ""
-        sockets_per_node = 2
+
+        sockets_per_node = self.options.sockets
+        if not sockets_per_node:
+            try:
+                proc_cpuinfo = open('/proc/cpuinfo').read()
+            except IOError as err:
+                self.log.debug("No info on sockets found; hardcoding to 2 (most likely)")
+                sockets_per_node = 2
```

better fail hard here, and mention to use `--sockets-per-node`?